Doc/library/functions.rst

```diff
@@ -928,22 +928,22 @@ are always available.  They are listed here in alphabetical order.
 
    Return an :term:`iterator` object.  The first argument is interpreted very
    differently depending on the presence of the second argument. Without a
-   second argument, *object* must be a collection object which supports the
+   second argument, *object* must be a collection object that supports the
    iteration protocol (the :meth:`__iter__` method), or it must support the
    sequence protocol (the :meth:`__getitem__` method with integer arguments
    starting at ``0``).  If it does not support either of those protocols,
    :exc:`TypeError` is raised. If the second argument, *sentinel*, is given,
    then *object* must be a callable object.  The iterator created in this case
    will call *object* with no arguments for each call to its
    :meth:`~iterator.__next__` method; if the value returned is equal to
-   *sentinel*, :exc:`StopIteration` will be raised, otherwise the value will
+   *sentinel*, :exc:`StopIteration` will be raised, otherwise, the value will
    be returned.
 
    See also :ref:`typeiter`.
 
    One useful application of the second form of :func:`iter` is to build a
    block-reader. For example, reading fixed-width blocks from a binary
-   database file until the end of file is reached::
+   database file until the end of the file is reached::
```

Comment:
This makes it arguably less readable, "end of file" is an [IT idiom](https://en.wikipedia.org/wiki/End-of-file).